### ISSUE TITLE
Proposition: Change UX for plugin unload in snapctl

### ISSUE THREAD

**Issue by danielscottt** [intelsdi-x/snap#608]
Currently, the way to unload a plugin via snapctl looks like this:

```sh
$ snapctl plugin unload -t collector -n foo -v 1
```

I propose just a single, positional argument:

```sh
$ snapctl plugin unload collector:foo:1
```

This issue arose from a discussion on #607.

**Comment by thomastaylor312** [comment]
@danielscottt +1 on this. It has always been a royal pain to unload a plugin and it is easy to forget one of the arguments.

**Comment by tiffanyfj** [comment]
Fixed by #616

**Comment by tiffanyfj** [comment]
Accidentally reopened it since I didn't realize merging my fix closed it...
